src/site/server/index.markdown

```diff
@@ -8,40 +8,168 @@ has-permalinks: false
 [API documentation](https://api.dartlang.org/apidocs/channels/stable/dartdoc-viewer/home).
 
 * [Tutorial: Write Command-Line Apps](/docs/tutorials/cmdline/)
 * [Tutorial: Write HTTP Clients & Servers](/docs/tutorials/httpserver/)
+* [Walkthrough of a working sample server][walkthrough] complete with
+  HTTP server, web sockets, logging, and routing
+
+### Get support
+
+Join our
+[cloud@dartlang.org](https://groups.google.com/a/dartlang.org/forum/#!forum/cloud)
+mailing list and chat with other engineers building
+server-side Dart apps.
+
+Post your "how-to" questions to
+[Stack Overflow](http://stackoverflow.com/tags/dart).
+
+And of course, we welcome [bug reports](http://dartbug.com/new).
+
+### Libraries
+
+There are lots of options for building server-side and command-line
+apps with Dart. Here are just some of our favorites:
+
+* [dart:io][dart_io_libs] - core functionality, such as files, directories,
+  sockets, HTTP, and lots more
+* [shelf][shelf]: middleware system for web servers, with a robust plugin system
+* [redstone][redstone]: server-side microframework for Dart
+* [MySQL driver][mysql]: connect to MySQL from Dart
+* [PostgreSQL driver][postgres]: connect to PostgreSQL from Dart
+* [args][args]: configurable parsing of command-line args
+* [http][http]: easily connect to HTTP resources
+* Check [pub.dartlang.org][pub] for more options and libraries, including
+ORM, Redis, Mongo, authentication, Google APIs, and more
+
+### Learn more
+
+Watch this video, recorded at the [Dart Developer Summit](/events/2015/summit/),
+about moving from Node.js to Dart.
+
+<iframe width="640" height="360" src="https://www.youtube.com/embed/NHsmiY0rFS8?list=PLOU2XLYxmsIIQorIS8gagUiMau9S84vZV&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
+
+Then, dive deep with our [articles](/articles/), examples,
+and other documentation.
+
 * [Articles on a variety of Dart subjects](/articles/),
    many written by Dart engineers
-* [Programmers Guide: Command-Line Apps](/docs/serverguide.html)&mdash;a
+* [Dart for the Server](/server/)&mdash;a
   collection of links to examples, tools, and other documentation
   of interest to server-side programmers
+* [Dart by Example][examples] has tons of small code snippets for servers
+  and consoles
 
-### Tell me more about the services that Dart supports.
-
-Check out the following for information about services that Dart supports. 
-
-* [Dart and Google Cloud Platform](google-cloud-platform/)
-* [Google APIs Client Libraries with Dart](/googleapis/)
-
-### Where can I host my Dart server?
+### Hosting Dart servers
 
 You have many options. Here are a few:
 
 * Google Cloud Platform; for details on Dart support, see
   [Dart and Google Cloud Platform](google-cloud-platform)
+* [Heroku](https://github.com/igrigorik/heroku-buildpack-dart)  
 * [Dartvoid](http://www.dartvoid.com/)
-* [Amazon Elastic Compute Cloud (EC2)](http://aws.amazon.com/ec2/)
 
+Of course, Dart runs nearly any cloud provider, such as
```

Comment:
I don't understand this sentence. How does Dart run a cloud provider? Why is it "of course"?